final/lightningd/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -e
+
+cp /testnet-config ${LIGHTNINGD_DATA}/testnet-config
+
+if [ $(echo "$1" | cut -c1) = "-" ]; then
```

Comment:
see comments in the bitcoind entrypoint

Reply:
so I just set the `lightnind` whenever we call without any argument rest is handled by setting user